README.md

```diff
@@ -1,2 +1,20 @@
 # marketing-accessibility
 Accessibility standards and search optimization
+
+Description:-
+Refactoring task. Consolidated many classes in CSS reducing duplicated code. Repaired links in navigation bar. Input new title. Replaced many element titles with semantic titles.
+
+Image of code:
```

Comment:
Maybe consider styling this as a heading using markdown syntax. See [link](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)